generateCarousel.js

```diff
@@ -0,0 +1,57 @@
+import { fetchData } from './getData.js'
+import { carouselNavButtonsEventListeners } from './script.js'
+
+function generateCarouselPhotos(item, index) {
+	const prevButton = document.getElementById('left-btn')
+	const elementClass =
+		index === 0
+			? 'carousel__image carousel__image--active fade'
+			: 'carousel__image fade'
```

Comment:
```suggestion
	const elementClass = `carousel__image fade${index === 0 ? 'carousel__image--active' : ''}`
```
Just a suggestion on different approach, feel free to choose whichever seems more readable to you. 😉